src/seedu/addressbook/AddressBook.java

```diff
@@ -205,12 +205,52 @@
 	 */
 	public static void main(String[] args) {
 		showWelcomeMessage();
-		processProgramArgs(args);
+//		processProgramArgs(args);
```

Comment:
Don't leave commented out code. Just go ahead and delete unused code. Now that we are using a revision control system, we can recover any deleted code later.